fluxcd-kustomize-mutating-webhook.yaml

```diff
@@ -0,0 +1,70 @@
+package:
+  name: fluxcd-kustomize-mutating-webhook
+  version: "0.5.0"
+  epoch: 0
+  description: A dynamic solution to patch FluxCD Kustomization resources, seamlessly integrating and federating substitution variables across multiple namespaces.
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/xunholy/fluxcd-kustomize-mutating-webhook
+      tag: kustomize-mutating-webhook-${{package.version}}
+      expected-commit: 694bac9d7eb73d827147ec0cb22c51cf4a31d5fe
+
+  - uses: go/build
+    with:
+      packages: ./cmd/webhook/
+      ldflags: |
+        -s -w
```

Comment:
I think we can remove these, they are default in our pipeline

Reply:
removed